tonic-web/Cargo.toml

```diff
@@ -26,6 +26,10 @@ tower-service = "0.3"
 tower-layer = "0.3"
 tracing = "0.1"
 
+[dependencies.axum]
+version = "0.8.1"
+optional = true
```

Comment:
we tend to use the normal toml style of `{ version, optional}` can you change this to that?